hv.c

```diff
@@ -49,7 +49,7 @@ holds the key and hash value.
 static const char S_strtab_error[]
     = "Cannot modify shared string table in hv_%s";
 
-#define HVhek_MASK	0xFF
+#define HVhek_STORAGE_MASK (HVhek_UTF8|HVhek_WASUTF8)
```

Comment:
In commit message: `s'(at least) types'(at least) 3 types'`

I note this is the same definition as `hv.h:HVhek_ENABLEHVKFLAGS`, do we want both of them?

Reply:
So...

I *think* so. But I thought quite a bit about it because it's not obvious what the right answer is. I changed the definition to this:

    #define HVhek_STORAGE_MASK (0xFF & ~HVhek_NOTSHARED)

which for the current 3 (used) bits is equivalent, and wrote a long comment to clarify what the 3 bits are used for, why we don't have any obvious single way to "group" them, and why it (probably) makes sense to have both `HVhek_ENABLEHVKFLAGS` and `HVhek_STORAGE_MASK`.

This is the commit whose message starts *Eliminate "masked_flags" from functions in hv.c*. After rebasing and editing, it is now 4f72a47f3a7931baf20d4d81a086b0f765e0bd62